### PR TITLE
Open source

### PR DESCRIPTION
**Pull request by JuanCaicedo**
@sandinmyjoints 

Set up for us to open source this module.

@swoloszynski @CarsonBanov

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Hoang Pham Huu <phamhuuhoang@gmail.com>
+Copyright (c) 2010-2016 Curiosity Media, Inc.
```
Comment (sandinmyjoints) [inline LICENSE]: I'm not sure we can just change the copyright like this. Can you spend 15 minutes googling to try to see how copyright works for forks?

File: README.md [hunk 2]
```diff
@@ -1,50 +1,53 @@
...
-    npm install --save ghost-s3-storage
+```bash
+$ npm install --save ghost-s3-compat
+```
 
-## Create storage module
+## Create a storage module
 
-Create index.js file with folder path 'content/storage/ghost-s3/index.js' (manually create folder if not exist)
+Create a file `content/storage/ghost-s3/index.js` (manually those folders if
```
Comment (sandinmyjoints) [inline README.md]: manually **create** those folders

File: package.json [hunk 3]
```diff
@@ -1,14 +1,14 @@
 {
-  "name": "ghost-s3-storage",
-  "version": "0.2.2",
-  "description": "Store Ghost's media at AWS S3 storage. Modified version of ghost-s3 (https://www.npmjs.org/package/ghost-s3) and ghost-aws-s3 (https://github.com/kimar/ghost-aws-s3) ",
+  "name": "ghost-s3-compat",
+  "version": "0.1.0",
```
Comment (sandinmyjoints) [inline package.json]: Per the first two points in http://semver.org/#faq, I think 1.0.0 might be appropriate. But you can make the final call.

**Comment by sandinmyjoints** [comment]
A few comments. BTY.

File: LICENSE (hunk 1 above)
Comment (JuanCaicedo) [inline LICENSE]: I found [this](http://programmers.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project). I'll update based on that.

**Comment by JuanCaicedo** [comment]
@sandinmyjoints Updated based on your comments, BTY

File: LICENSE (hunk 1 above)
Comment (sandinmyjoints) [inline LICENSE]: Great.

File: README.md [hunk 4]
```diff
@@ -1,50 +1,53 @@
...
-
-Restart app then test upload new image in blog post. Image will be store at newly S3 bucket.
+You can add `assetHost` to your config to specify a virtual host url. For more
+information, [read this section](http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html)
+in the AWS docs.
 
 ## Copyright & License
 
-Copyright (c) 2015  Hoang Pham Huu <phamhuuhoang@gmail.com>
+Copyright (c) 2010-2016 Curiosity Media, Inc.
```
Comment (sandinmyjoints) [inline README.md]: Apply the same copyright change here.

**Comment by sandinmyjoints** [comment]
One more comment. Approved once it's addressed.

**Pull request merged**